tekton/src/pipelines/taskdefs/fvt-core/phase2.yml.j2

```diff
@@ -1,12 +1,12 @@
 # Only start phase 2 tests after all suites in phase 1 are complete
 
-# Core FVT 2.1 coreapi-mobileapi ~5m
-- name: fvt-coreapi-mobileapi
+# Core FVT 2.0 coreapi-usermgmt-v3 ~30m
```

Comment:
So, now phase 2 will be like:

After run `phase1`, the following tests will run in parallel:
- fvt-coreapi-usermgmt-v3
- fvt-coreapi-dependencies
- fvt-coreapi-messages

And after `fvt-coreapi-usermgmt-v3`, the following wil run parallel:
- fvt-coreapi-mobileapi
- fvt-coreapi-other
- fvt-coreapi-utilities
- fvt-coreapi-uiresources

Is that is the idea, looks good 👍